CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.2
+
+- `Layout/FirstHashElementIndentation` is deprecated. Use `Layout/FirstHashElementIndentation` instead.
```

Comment:
These two seems identical? 🤔

Reply:
🤦‍♂

Reply:
Will fix when I'm done testing how this PR will affect Omoikane